tests/certification/embedded/embedded.go

```diff
@@ -50,7 +50,7 @@ const (
 	maxConcurrency      = -1
 	enableMTLS          = false
 	sentryAddress       = ""
-	maxRequestBodySize  = 4
+	maxRequestBodySize  = 4 << 20
```

Comment:
what does this magic number do?

Reply:
see here in that pr where they are using similarly 
<img width="712" alt="image" src="https://github.com/dapr/components-contrib/assets/25728646/0b84d430-af93-4e2f-85c9-2d0eb8d76361">

Reply:
@berndverst lmk if you need further clarification